### PR TITLE
update master

### PR DESCRIPTION
**Pull request by JeffGreiner-eaton**
Fixes BLUI-4594.

#### Changes proposed in this Pull Request:

- update master branch with blui-pr-actions plus the other changes
-
-

#### Screenshots / Screen Recording (if applicable)

-

#### To Test:

- once this is merged we can open a PR to master to verify labels and comment

#### Any specific feedback you are looking for?

-

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -96,17 +85,10 @@ workflows:
               only:
                 - master
                 - dev
-      - tag:
```
Comment (joebochill) [inline .circleci/config.yml:99]: Why did we remove the tag job?

Comment (daileytj) [reply .circleci/config.yml:99]: Is that handled by the tagging workflow in the .github/workflows folder?

Comment (JeffGreiner-eaton) [reply .circleci/config.yml:99]: it was changed back in Dec 28, 2022 and looks like the tagging.yml is used instead. https://github.com/etn-ccis/blui-react-auth-shared/blob/dev/.github/workflows/tagging.yml
Do we want to cleanup the circleCI config for items not used now or fly with this?

Comment (joebochill) [reply .circleci/config.yml:99]: Yep, I'm a goober. I missed the part where we moved it into GH Actions.

File: CHANGELOG.md [hunk 2]
```diff
@@ -5,6 +5,17 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## v4.0.0 (Unreleased)
```
Comment (joebochill) [inline CHANGELOG.md:8]: A merge into master implies that we're ready to publish. There are a couple of enhancements that are in here that we put in as workarounds for Fiji. We made them a 4.0.0-alpha package (currently what's in package.json) anticipating that there would be breaking changes as part of the refactoring work.

We ended up detaching from shared-auth altogether on the React side and I anticipate we will probably do the same when we get to RN. So these couple of changes can/should get published as a minor version instead (3.8.0)

**Review by joebochill — APPROVED** [review]

**Review by daileytj — APPROVED** [review]

**Pull request merged**